1주차/5622.py

```diff
@@ -0,0 +1,28 @@
+#5622
+##걸린 시간 = 다이얼 숫자+1
+
+A=input("")
+B=list(A)
+num=0
+sum=0
+for i in range(0,len(B)):
+  if (ord(B[i])>= ord('A') and ord(B[i])<=ord('C')):
```

Comment:
```suggestion
  if 'A' <= B[i] <= 'C':
```

1. 굳이 ord함수 써서 숫자로 바꾸지 않아도 문자열끼리 직접 비교 가능합니다.
2. 파이썬은 어떤 범위 내에 있는지 확인하는 조건을 이렇게 수학 조건식처럼 한번에 쓸 수 있어요.
3. 파이썬 if문은 조건을 괄호로 감싸주지 않아도 됩니다